docs/src/docs/usage/install/index.mdx

```diff
@@ -24,17 +24,17 @@ use deprecated option `--enable-all` and a new linter is added or even without `
 
 It's highly recommended to install a specific version of golangci-lint available on the [releases page](https://github.com/golangci/golangci-lint/releases).
 
-Here is the recommended way to install golangci-lint {.LatestVersion}:
```

Comment:
I am not so sure if we are supposed to change these files. As I think these are templates. Might work for _this commit_ only, but the next run will not change anything right ?

Reply:
I reverted this changes, otherwise the `Netlify` fails

Reply:
I run ` make expand_website_templates` and it changed the templates.
The problem here is that the Netlify build the documentation with old version, but I think this happens because the assets is not merged yet. Let merge this PR and check what will happen :-)